docs/getting-started/overview.md

```diff
@@ -0,0 +1,62 @@
+# Getting Started
+
+Welcome to STAC Server! This guide will help you understand what STAC Server is and how to get started.
+
+## What is STAC?
+
+The **[SpatioTemporal Asset Catalog (STAC)](https://stacspec.org/)** specification provides a common language to describe geospatial information, making it more easily discoverable and accessible. STAC enables interoperability between systems and allows users to search across multiple catalogs.
+
+## What is STAC Server?
+
+STAC Server is a production-ready implementation of the [STAC API specification](https://github.com/radiantearth/stac-api-spec). It provides:
+
+- **RESTful API**: Search and access spatiotemporal data via HTTP
+- **Elasticsearch Backend**: Scalable indexing and search capabilities
+- **Serverless Architecture**: Runs on AWS Lambda with API Gateway
+- **Extensible**: Supports multiple [STAC extensions](https://stac-api-extensions.github.io/) out of the box
+
+## Key Concepts
+
+### Collections
+
+Collections group related Items together. Each Collection describes:
+
+- Spatial and temporal extents
+- Common properties shared by Items
+- Available assets and their formats
+- Licensing and provider information
+
+### Items
+
+Items represent individual spatiotemporal assets (e.g., satellite scenes, drone imagery). Each Item contains:
+
+- Unique identifier
+- Geometry (footprint)
+- Datetime(s)
+- Properties (metadata)
+- Assets (links to actual data files)
+
+### Catalog
+
+The root endpoint that provides links to all Collections and search capabilities.
+
+## Prerequisites
+
+Before deploying STAC Server, ensure you have:
+
+- **AWS Account** with appropriate permissions
+- **Node.js** 18.x or later
```

Comment:
Something like "the version defined in .nvmrc" may be better here. Currently it's 22, but obviously will change in future